expat/coverage.sh

```diff
@@ -41,6 +41,10 @@ _configure() {
     ${unicode_enabled} \
             && configure_args+=( CPPFLAGS='-DXML_UNICODE -DXML_UNICODE_WCHAR_T' )
 
+    if [ "${xml_attr_info}" = true ]; then
+       configure_args+=( --enable-xml-attr-info )
```

Comment:
Indentation seems off here (3 versus 4 spaces).

Reply:
Addressed review comments in latest patchset